go/e2e_test_utils/launcher.go

```diff
@@ -50,7 +52,7 @@ func LaunchTests(testFunctions []func(context.Context, *sync.WaitGroup, chan *ju
 }
 
 // RunTestsAndOutput runs tests by the test framework and output results to given file
-func RunTestsAndOutput(testFunctions []func(context.Context, *sync.WaitGroup, chan *junitxml.TestSuite, *log.Logger, *regexp.Regexp, *regexp.Regexp, *testconfig.Project),
+func RunTestsAndOutput(testFunctions []func(context.Context, *sync.WaitGroup, chan *junitxml.TestSuite, *log.Logger, *regexp.Regexp, *regexp.Regexp, *testconfig.Project, map[string]string),
```

Comment:
I believe LaunchTests and RunTestsAndOutput have been used by other projects (os-config tests for example). Please add a new Interface, RunTestsAndOutputWithVars